### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by ukrocks007**
- Setup link documentation
- Screenshots updated
- Audit logs admin portal page shifted
- Added retraced architectural diagram
- retraced self-host and deploy sections merged
- Removed auth0 environment variables

### CODE REVIEW

**Review by devkiran — CHANGES_REQUESTED** [review]
- I think there are some missing headings in the docs. Here is a screenshot of the same. Please take a look at it. doc-1

- Can we use the Markdown syntax for the images? It will be easier to maintain and will be consistent with the rest of the docs. Docusaurus add zoom functionality to the images when we use the Markdown syntax. Not sure if we can do the same with the HTML syntax.

- It would better to use "Admin Portal" instead of "Admin UI" in the docs. (/docs/admin-portal/audit-logs). We are using "Admin Portal" other places as well.

**Comment by ukrocks007** [comment]
> * I think there are some missing headings in the docs. Here is a screenshot of the same. Please take a look at it. doc-1
> * Can we use the Markdown syntax for the images? It will be easier to maintain and will be consistent with the rest of the docs. Docusaurus add zoom functionality to the images when we use the Markdown syntax. Not sure if we can do the same with the HTML syntax.
> * It would better to use "Admin Portal" instead of "Admin UI" in the docs. (/docs/admin-portal/audit-logs). We are using "Admin Portal" other places as well.

Fixed

File: docs/directory-sync/setup-link.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
```
Comment (deepakprabhakara) [inline docs/directory-sync/setup-link.md:1]: This should go into Admin Portal section right?

File: docs/jackson/setup-link.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
```
Comment (deepakprabhakara) [inline docs/jackson/setup-link.md:1]: This should go into Admin Portal section right?

File: docs/retraced/sbom.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+[SBOM](https://en.wikipedia.org/wiki/Software_bill_of_materials) is a list of components in a piece of software. It is like a list of ingredients of a product. Is it quickly becoming a standard for software supply chain risk and the ecosystem is growing with tools that help you verify, scan and monitor all your dependencies.
+
+## Report Standards
+
+SBOM reports primarily use [SPDX](https://en.wikipedia.org/wiki/Software_Package_Data_Exchange) & [CycloneDX](https://cyclonedx.org/) standards.
+
+## Reports
+
+**Note:** This is supported for all versions >=0.3.8
```
Comment (deepakprabhakara) [inline docs/retraced/sbom.md:11]: >=v1.5.0

**Review by deepakprabhakara — APPROVED** [review]

**Pull request merged**